MANIFEST.in

```diff
@@ -14,8 +14,8 @@ graft requirements
 graft tests
 global-exclude *.pyc
 global-exclude *.cache
-exclude propcache/*.c
-exclude propcache/*.html
-exclude propcache/*.so
-exclude propcache/*.pyd
+exclude src/propcache/*.c
+exclude src/propcache/*.html
+exclude src/propcache/*.so
+exclude src/propcache/*.pyd
```

Comment:
Should have been done in the `src` PR.  Should be moved to a separate PR

Reply:
I think that `.pyd` shouldn't be excluded, though. We don't have it, but it's one of the Cython source types IIRC...

Reply:
I'm not sure if `.so` or `.pyd` should be excluded actually.

`html` and `c` from cython definitely should be

Reply:
https://github.com/aio-libs/propcache/pull/29